docs/Hooks.md

```diff
@@ -131,7 +131,15 @@ fastify.addHook('onClose', (instance, done) => {
 ```
 <a name="scope"></a>
 ### Scope
-Except for `'onClose'` all the hooks are encapsulated this means that you can decide where your hooks should run by using `register` as explained in the [plugins guide](https://github.com/fastify/fastify/blob/master/docs/Plugins-Guide.md).
+Except for `'onClose'` all the hooks are encapsulated this means that you can decide where your hooks should run by using `register` as explained in the [plugins guide](https://github.com/fastify/fastify/blob/master/docs/Plugins-Guide.md). If you pass a function, that function is bound to the right Fastify context and from there you have full access to the Fastify api.
+
+```js
+fastify.addHook('onRequest', (req, res, next) => {
```

Comment:
This is not correct, you should not use an arrow function to access the correct this :D

Reply:
:D

Reply:
Done